test/integration/location_js_test.rb

```diff
@@ -13,9 +13,11 @@ class LocationJSTest < IntegrationTestWithJavascript
     assert_new_button(locations_path, "New Location", new_location_path)
     fill_in "location_name", :with => "Raleigh"
     click_button "Submit"
-    assert_current_path step2_location_path(Location.unscoped.order(:id).last)
+    # wait for submit to finish
+    page.all('.breadcrumb-item div', text: 'Edit Raleigh')
```

Comment:
Just a question: I've noticed that there are two ways on how to wait for submit to finish. Is there any benefit from using this approach instead of https://github.com/theforeman/foreman/blob/develop/test/integration/hostgroup_js_test.rb#L26-L28? Are they kinda identical?

Reply:
1. I didnt see this way
2. I tried it and got this error in half of the test runs
```
retry 'test_0002_create new page when some hosts are and assigned a location and click Proceed to Edit' count: 1,  msg: Timeout::Error: execution expired
    test/integration_test_helper.rb:180:in `wait_for'
```

Reply:
> I tried it and got this error in half of the test runs

anxiety++

Okay then, let's stick with this way